protected/humhub/modules/user/commands/UserController.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2020 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\installer\commands;
+
+use Yii;
+use yii\console\Controller;
+use yii\console\ExitCode;
+use yii\helpers\Console;
+use yii\base\Exception;
+use humhub\modules\admin\models\UserSearch;
+use humhub\modules\user\models\User;
+use humhub\modules\user\models\Password;
+use humhub\modules\user\models\Group;
+
+/**
+ * Console User management
+ * 
+ * Example usage:
+ *   php yii user/create-admin-account
+ *   php yii user/create-account "$HUMHUB_USERNAME" "$HUMHUB_EMAIL" "$HUMHUB_PASSWORD"
+ *   php yii user/set-user-password "$HUMHUB_USERNAME" "$HUMHUB_PASSWORD"
+ * 
+ * @author Luke
+ * @author Michael Riedmann
+ * @author Mathieu Brunot
+ */
+class UserController extends Controller
+{
+
+    /**
+     * Creates a new user account and adds it to the admin-group
+     */
+    public function actionCreateAdminAccount($admin_user', $admin_email, $admin_pass,
```

Comment:
@madmath03 You added an unneeded  `'` which will cause an error

Reply:
Thanks @ArchBlood for the quick review ;)